### PR TITLE
Add SKR test for creating more than 10 bindings

### PR DESCRIPTION
**Pull request by MarekMichali**
**Description**

Changes proposed in this pull request:

- Add SKR test for creating more than 10 bindings

**Related issue(s)**

See #1204

### CODE REVIEW

File: testing/e2e/skr/skr-binding-test/index.js [hunk 1]
```diff
@@ -162,6 +162,32 @@ describe('SKR Binding test', function() {
     }
   });
 
+  it('Should not allow creation of more than 10 SKR bindings', async function() {
+    errorOccurred = false;
+    count = 0;
+    while (!errorOccurred && count < 13) {
```
Comment (MarekMichali) [inline testing/e2e/skr/skr-binding-test/index.js:168]: We don't know how many bindings have been created in the previous test before we start this one.

Comment (ralikio) [reply testing/e2e/skr/skr-binding-test/index.js:168]: Could you add such comment in the code?

File: testing/e2e/skr/skr-binding-test/index.js [hunk 2]
```diff
@@ -162,6 +162,32 @@ describe('SKR Binding test', function() {
...
+      bindingID = Math.random().toString(36).substring(2, 18);
```
Comment (ralikio) [inline testing/e2e/skr/skr-binding-test/index.js:169]: Could we replace it with uuid generation?

Comment (MarekMichali) [reply testing/e2e/skr/skr-binding-test/index.js:169]: No problem, I will replace it also in the existing parts of code.

**Review by ralikio — APPROVED** [review]

**Pull request merged**